bylaws.md

```diff
@@ -150,6 +150,8 @@ Vacancies on the board may be filled by approval of the board or, if the number
 
 A person elected to fill a vacancy as provided by this Section shall hold office until the next annual election of the board of directors or until his or her death, resignation, or removal from office.
 
+If a director is absent from three regular meetings all in separate months in a four month period, a vote will automatically be held in the next meeting of directors to remove said director from the board.
```

Comment:
We should add something about prior consent. If someone goes on parental leave or gets in a serious accident, we shouldn't automatically boot them. Both of these have happened before. 

> If a director is absent without prior consent from three monthly meetings within four months, a vote will automatically be initiated at the subsequent meeting of directors to decide on the possible removal of said director from the board.

(I ChatGPT'ed and modified this so suggest away)

Reply:
Hmm I meant for this to require us to vote, not automatically remove. It's up to the board to make the decision based on the information present. In those cases, it'd likely result in a vote to reaffirm.


Edit: switched to mobile from email and put the comment on the thread.

Reply:
Rereading it you're right. There's a heavy implication. Good catch.

I should also account for what happens if we vote to reaffirm. How long does that immunity last? (rhetorical unless people have opinions)